starters/nestjs-bullmq-postgresql/package.json

```diff
@@ -12,6 +12,7 @@
 		"start:dev": "nest start --watch",
 		"start:debug": "nest start --debug --watch",
 		"start:prod": "node dist/main",
+		"start:all": "yarn infrastructure:start && yarn start:dev",
```

Comment:
Do we have a naming convention for that kind of scripts? Does `start:all` follow that?